src/Common/Tooltip/Tooltip.tsx

```diff
@@ -1,24 +1,26 @@
-import { ITooltipHostStyles, TooltipHost } from "@fluentui/react";
+import { IButtonStyles, IconButton, ITooltipHostStyles, TooltipHost } from "@fluentui/react";
 import { useId } from "@fluentui/react-hooks";
 import * as React from "react";
-import InfoBubble from "../../../images/info-bubble.svg";
 
 const calloutProps = { gapSpace: 0 };
 const hostStyles: Partial<ITooltipHostStyles> = { root: { display: "inline-block" } };
 
 export interface TooltipProps {
   children: string;
 }
+
+const iconButtonStyles: Partial<IButtonStyles> = { root: { marginBottom: -3 } };
```

Comment:
Use `vertical-align` instead of hard-coded `margin-bottom` value to properly align the icons.

Reply:
done